examples/exec-healthz/README.md

```diff
@@ -0,0 +1,12 @@
+### SimpleHealthzServer
+
+SimpleHealthzServer is a sidecar container meant to serve as an exec-over-http bridge.
```

Comment:
This does sync. We could do async and pipe through another -period flag, and default it to 10s or something. That might conflict with a higher lever poll interval, when we have one.